Kickstarter-iOS/Views/Cells/DiscoveryProjectCardCell.swift

```diff
@@ -354,7 +386,13 @@ final class DiscoveryProjectCardCell: UITableViewCell, ValueCell {
     _ = ([self.backersCountIconImageView, self.backersCountLabel], self.backersCountStackView)
       |> ksr_addArrangedSubviewsToStackView()
 
-    _ = ([self.goalPercentFundedStackView, self.backersCountStackView], self.projectInfoStackView)
+    let spacer = UIView(frame: .zero)
+      |> UIView.lens.contentHuggingPriority(for: .horizontal) .~ UILayoutPriority(rawValue: 249)
```

Comment:
`.defaultLow` didn't work here?

Reply:
The default setting for `contentHuggingPriority` on views is `defaultLow`, so this needed something that was just below that value.